app/views/fae/shared/_form_header.html.slim

```diff
@@ -10,30 +10,35 @@ ruby:
   title              ||= "#{params[:action]} #{header_as_string}".titleize
   breadcrumb           = !local_assigns[:breadcrumbs].eql?(false)
   languages          ||= false
+  hide_flash         ||= false
```

Comment:
This reads as a double negative, I would prefer `show_flash` with the default being true. Of course you can't pass in `false` with the standard or equauls, I think you have to do something like:

```ruby
show_flash = true if show_flash.nil?
```